### ISSUE TITLE
Internal Server Error in /api/update_dns

### ISSUE THREAD

**Issue by adrgs** [adrgs/requestrepo#37]
Just noticed, need to fix this

**Comment by adrgs** [comment]
```
  File "/usr/local/lib/python3.10/site-packages/fastapi/routing.py", line 191, in run_endpoint_function
    return await dependant.call(**values)
  File "/app/app.py", line 119, in update_dns
    await redis.set(f"dns:{subdomain}", json.dumps(final_records))
  File "/usr/local/lib/python3.10/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.10/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.10/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.10/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DnsRecord is not JSON serializable
```

**Comment by adrgs** [comment]
Fixed in #38
